package.json

```diff
@@ -15,13 +16,18 @@
   },
   "homepage": "https://github.com/lmucs/social-feed-calendar#readme",
   "dependencies": {
+    "assert": "^1.3.0",
     "body-parser": "^1.13.3",
     "coffee-script": "^1.10.0",
     "dotenv-with-overload": "^0.5.1",
     "express": "^4.13.3",
     "jwt-simple": "^0.3.1",
+    "mocha": "^2.3.3",
     "morgan": "^1.6.1",
     "pg": "^4.4.2",
-    "request": "^2.61.0"
+    "request": "^2.61.0",
+    "should": "^7.1.0",
+    "sinon": "^1.17.1",
+    "supertest": "^1.1.0"
   }
```

Comment:
[Best practices for package.json](https://docs.npmjs.com/files/package.json) say the following: "Please do not put test harnesses or transpilers in your dependencies object." Please siphon off the test stuff into the `devDependencies` section.